app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -124,6 +135,43 @@ def get_images
         process_collection(images, :vms) { |image| parse_image(image) }
       end
 
+      def get_security_groups
+        security_groups = gather_data_for_this_region(@nsg)
```

Comment:
@djberg96 
Yea if the method is being called in other places it makes sense to remove it at line 139.